examples/server/server.c

```diff
@@ -2320,6 +2337,12 @@ THREAD_RETURN WOLFSSL_THREAD server_test(void* args)
             quieter = 1;
             break;
 
+#ifdef WOLFSSL_DUAL_ALG_CERTS
+        case 267:
```

Comment:
Magic number - use a define.

Reply:
Would be inconsistent.  None of the other cases in this switch use defines.